src/main/java/frc/robot/RobotContainer.java

```diff
@@ -52,6 +53,7 @@ public class RobotContainer {
   private final ShooterSubsystem shooterSubsystem;
   private final TriggerSubsystem triggerSubsystem;
   private final TurretSubsystem turretSubsystem;
+  private final ClimberSubsystem climberSubsystem;
```

Comment:
Need to add some button mappings

Reply:
Kian is working on that in a separate branch.